test/mixin/entity-mixin-test.js

```diff
@@ -97,10 +97,10 @@ describe('d2l-organization-name', () => {
 					const semesterNameDirect = component.shadowRoot.getElementById('semester-name-direct');
 					const semesterName = component.shadowRoot.getElementById('semester-name').shadowRoot;
 
-					expect(name.innerHTML.replace(/<!---->/g, '')).to.be.equal('Course Name');
-					expect(code.innerHTML.replace(/<!---->/g, '')).to.be.equal('SCI100');
-					expect(semesterNameDirect.innerHTML.replace(/<!---->/g, '')).to.be.equal('Semester Name');
-					expect(semesterName.innerHTML.replace(/<!---->/g, '')).to.be.contains('Semester Name');
+					expect(name.innerHTML.replace(/<!--.*-->/g, '')).to.be.equal('Course Name');
```

Comment:
Lit 2 adds comment nodes with numbered contents for performance reasons. Example comment: 
`<!-- lit$6325699808$-->`